swiiny/create-nextjs-dapp
# Chore/pump packages

# Pull Request Template

## Description

Key updates include pumping all templates packages. It also includes fixing an MUI warning due to the usage of `spacing` keyword in the DefaultTheme. 

## Type of Change


## Steps to Test

1. Check the updated template packages for any inconsistencies or issues.
2. Test the spacing in the MUI to ensure the fix is effective.
3. Verify the repository URL updates for correctness.
4. Review the applied VSCode settings for any potential conflicts or improvements.

## Expected Behavior

The updates are expected to enhance the functionality and reliability of the `create-nextjs-dapp` templates, ensuring smoother development experiences. The fix in MUI spacing keyword usage should resolve warnings on server, and the repository URL updates should direct to the correct locations.

## Additional Information

Recent commits focused on updating template packages and fixing a specific MUI keyword error. The repository's overall health and consistency were also addressed by updating URLs and applying recommended VSCode settings.

## Checklist

- [x] I have commented my code, especially in areas that may be hard to understand for others.
- [ ] I have added tests that demonstrate my changes are effective or that my feature works.
